controllers/cephcluster_controller.go

```diff
@@ -32,22 +38,61 @@ type CephClusterReconciler struct {
 	client.Client
 	Log    logr.Logger
 	Scheme *runtime.Scheme
+	cc     *hypersdsv1alpha1.CephCluster
 }
 
 // +kubebuilder:rbac:groups=hypersds.tmax.io,resources=cephclusters,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=hypersds.tmax.io,resources=cephclusters/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups="",resources=pods;configmaps,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets;replicasets;statefulsets,verbs=get;list;watch;create;update;patch;delete
 
+// Reconcile reads that state of the cluster for a CephCluster object and makes changes based on the state read
 func (r *CephClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
 	_ = r.Log.WithValues("cephcluster", req.NamespacedName)
 
-	// your logic here
+	cachedCluster := &hypersdsv1alpha1.CephCluster{}
+	if err := r.Client.Get(context.TODO(), req.NamespacedName, cachedCluster); err != nil {
+		if errors.IsNotFound(err) {
+			return reconcile.Result{}, nil
+		}
+		return reconcile.Result{}, err
+	}
+	r.cc = cachedCluster.DeepCopy()
+
+	// TODO: error state
+	if err := r.syncConfigMap(); err != nil {
+		return ctrl.Result{}, err
+	}
+	if err := r.syncCephConfigMap(); err != nil {
```

Comment:
What about `syncAccessConfig()` or `syncPostInstallConfig()` ?

Reply:
I think `syncAccessConfig()` is better.